design-proposals/tdx_snp.md

```diff
@@ -0,0 +1,342 @@
+# Overview
+
+This proposal deals with the enablement of the following technologies: Intel
+TDX & AMD Secure Encrypted Virtualization with Secure Nested Paging (SEV-SNP)
+in kubevirt. This enablement includes the support for creating confidential VMs
+by using TDX & SEV-SNP by encrypting VM memory. For Intel TDX, This proposal
+relies on the attestation by using trustee client and the trustee operator.
+For AMD, SEV is currently supported in KubeVirt, and this VEP proposes to build
+on top of this feature and integrates SEV-SNP support.
+
+## Motivation
+
+Organizations are now asking for workloads to include the strongest
+confidential computing security guarantees.
+
+For AMD, KubeVirt supports SEV and SEV-ES for memory encryption, but lacks
+SEV-SNP enhanced memory integrity protection.
+
+For Intel, Add support for TDX technology for in-use memory encryption of
+confidential guests.
+
+## Proposal
+
+- Enable TDX VM Deployment: Provide users the ability to deploy VMs that are
+  utilizing INTEL TDX technology.
+- Enable TDX VM Blob Generation: Provide users the ability to generate the blob
```

Comment:
```suggestion
- Enable TDX VM Guote Generation: Provide users the ability to generate the quote
```

Reply:
I remove the line about quote generation.